CONTRIBUTING.md

```diff
@@ -26,8 +26,8 @@ Composer.
 
 The test suite may be executed with:
 
-```
-$ vendor/bin/simple-phpunit
+```console
```

Comment:
`console` does not exist - the "language" you're looking for is [`Shell`](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/lib/linguist/languages.yml#L6393)

Reply:
It's an [alias](https://github.com/github-linguist/linguist/blob/e855ef2b6f90c34074061a2e17acbe853e61b483/lib/linguist/languages.yml#L6482).

Reply:
Ah, good one!